### PR TITLE
Refactor rama

### PR DESCRIPTION
**Pull request by JoshuaNg2332**
Fixing the signup page to allow the page to create a post request and be redirected to the welcome page without showing the email and password in the URL.

Also added in the role over message for password formatting.

### CODE REVIEW

File: app/views/signup/index.html.erb [hunk 1]
```diff
@@ -1 +1,86 @@
-<h1> Welcome to the signup page </h1>
\ No newline at end of file
+<!DOCTYPE html>
```
Comment (JoshuaNg2332) [inline app/views/signup/index.html.erb:1]: Need to figure out how to get this to read from the CSS file.

**Review by jamesAforster — COMMENTED** [review]
Seems like we've replicated the CSS stuff in html.erb as well? So can probably just remove those lines in html.erb.

File: app/views/signup/template.html [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
```
Comment (JoshuaNg2332) [inline app/views/signup/template.html:1]: This is purely for adding the html format into the .erb files as they aren't recognised by vscode as html files.

**Review by jamesAforster — COMMENTED** [review]
Looks good to me!

**Pull request merged**